pkg/network/setup/netpod/netpod.go

```diff
@@ -276,9 +276,8 @@ func (n NetPod) composeDesiredSpec(currentStatus *nmstate.Status) (*nmstate.Spec
 			}
 		case iface.SRIOV != nil:
 		case iface.Binding != nil:
+		// Passt is removed in v1.3. This scenario is tracking old VMIs that are still processed in the reconcile loop.
 		case iface.DeprecatedPasst != nil:
-			spec.LinuxStack.IPv4.PingGroupRange = []int{107, 107}
-			spec.LinuxStack.IPv4.UnprivilegedPortStart = pointer.P(0)
```

Comment:
This looks wrong. We are supposed to keep the reconcile intact so existing running VMs will not be affected.

Reply:
From what I have tested with passt we can remove the remaining and its not causing disruptions.
From now I am reverting this change.
Hopefully we can get rid of the remaining in a follow up PR.
 
Done.